app/src/components/common/Button.vue

```diff
@@ -21,7 +21,7 @@ export default class Button extends Vue {
     validator: (value: string) => [ 'primary', 'secondary' ].includes(value)
   })
   styleType!: string
-  @Prop({ required: true }) title!: string
+  /*@Prop({ required: true }) title!: string*/
```

Comment:
Did you leave this commented code on purpose?

Reply:
Thanks for reminding, fixed!

Reply:
Thanks for reminding, fixed!